README.md

```diff
@@ -1,35 +1,39 @@
 # DACSspace
 
-A simple Python script to evaluate your ArchivesSpace instance for DACS compliance.
-
-This repository is currently under development!
+A simple Python script to evaluate your ArchivesSpace instance for DACS [single-level minimum](http://www2.archivists.org/standards/DACS/part_I/chapter_1) required elements. 
 
 ## Requirements
 
-Add software operating systems, programming languages or libraries which are required by this project as an unordered list like this:
-
 *   Python (written for and tested on 2.7).
 *   ConfigParser
 *   Requests module
 
 ## Installation
 
-Install requirements, then run `setup.py` to create a configuration file which will allow you to connect to your ArchivesSpace instance.
+Download [Python](https://www.python.org/downloads/)
+
+Add Python to your [PATH variable](https://docs.python.org/2/using/windows.html)
```

Comment:
This is only necessary if you're using Windows, so it would be good to specify that!